### DISCUSSION TITLE
How to use multiple guards in Middleware?

### DISCUSSION THREAD

**Discussion by carveybunt** [spatie/laravel-permission#1869] [Q&A]
I can't get how to use multiple guards in Middleware in the official document every version. If that is my need, how do I? The project can support?

**Comment by erikn69** [comment]
I think that is automatic, when user log in, he use a guard, that guard is used for the package

```php
Route::group(['middleware' => ['auth:consumer,member'] ], function(){
     Route::get('/home', 'HomeController@index');
});
```
